client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -16,6 +16,36 @@ import { withSelect } from '@wordpress/data';
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import { DEFAULT_POSTS, MAX_POSTS_TO_SHOW } from './constants';
 
+class MockupPostEdit extends Component {
```

Comment:
Since this class only implements the render function we could further reduce it to just a function.

Reply:
True. But this is also a question of sticking with conventions and accounting for the class being more extendible.

Reply:
We can turn it into a class later when needed (e.g. when adding lifecycle events) — better keep them as simple functions as long as class isn't needed. That pattern has been our convention. :-)